src/renderers/simple.js

```diff
@@ -162,12 +164,12 @@ export class Simple extends Renderer {
    * @private
    */
   layoutEdge_(edge, container, graph) {
-    const sourceNode = graph.getNode(edge.source);
-    const targetNode = graph.getNode(edge.target);
+    const sourceNode = edge.source;
+    const targetNode = edge.target;
```

Comment:
Add ternary. This will become a problem with future layouts that don't change these to nodes.